metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -206,14 +206,16 @@ def _register_job_definition(
                 k, v = each_log_option.split(":", 1)
                 log_options_dict[k] = v
 
-        job_definition["containerProperties"]["logConfiguration"] = {
-            "logDriver": log_driver if log_driver is not None else "awslogs",
-            "options": log_options_dict
-            if log_options is not None
-            else {
-                "awslogs-group": "aws/batch/job",
```

Comment:
was there a reason for removing this as the default?

Reply:
yeah, please refer to https://github.com/Netflix/metaflow/issues/1776

TLDR: AWS doesn't like when we supply the default values explicitly and not supplying is equivalent to supplying them implicitly anyway...

so it does result into using `aws/batch/job` even if we don't supply it...
while supplying it is not allowed because it is AWS specific and not custom...